### ISSUE TITLE
Training loss spike in put_bottles_dustbin task

### ISSUE THREAD

**Issue by kianyale** [HorizonRobotics/RoboOrchardLab#18]
Hi Horizon Robotics Team,

Thank you for your excellent open-source baseline for RoboTwin!

During training on the **put\_bottles\_dustbin** task, I noticed that the training loss suddenly increases sharply after reaching a certain step, then oscillates, and finally stabilizes at a relatively large value. I’d like to ask whether you have also observed this phenomenon. If so:

1. Does it affect the performance?
2. Have you figured out the reason behind this phenomenon?

I’ve attached the loss curve below. Looking forward to your response.

Best regards,
Qingyu

<img width="545" height="404" alt="Image" src="https://github.com/user-attachments/assets/6f424a0d-2977-4c9b-a61f-274315f81066" />

**Comment by Fanqyu** [comment]
Thank you for your reply!
Could you please suggest a reasonable range for the weight? The original loss_depth weight is set to 1.

**Comment by linxuewu** [comment]
0.1~0.3 might be a good choice.
